esm_runscripts/oasis.py

```diff
@@ -353,8 +355,10 @@ def prepare_restarts(self, restart_file, all_fields, model, config):
                 os.system("ncwa -O -a time onlyonetimestep.nc notimestep_" + field + ".nc")
                 filelist += "notimestep_" + field + ".nc "
                 print (filelist)
-        print("cdo merge " + filelist + " " + restart_file )#+ enddate)
-        os.system("cdo merge " + filelist + " " + restart_file )# + enddate)
+        # MA: -O flag added to overwrite oasis restart files in case oasis creats them
+        # before (i.e. when using LOCTRANS)
+        print("cdo -O merge " + filelist + " " + restart_file )#+ enddate)
```

Comment:
Maybe a little notification could be printed here. Such as 

```python
if os.path.isfile(<path_to_file> + filelist):
    print(f"{filelist} already exits, overwriting")
    ...    
```

Reply:
Thanks, I just added your suggestion